### DISCUSSION TITLE
426

### DISCUSSION THREAD

**Discussion by giscus** [Shane-Park/markdownBlog#170] [Comments]
# 426

문제 서버에서 javax email을 활용해 구글 이메일을 전송 할 때 아래와 같은 에러가 발생 했습니다. org.springframework.mail.MailSendException: Mail server connection failed; nested exception is javax.mail.MessagingException: Could not convert socket to TLS; nested exception is: javax.net.ssl.SSLHandshakeException: No appropriate protocol (protocol is disabled or cipher suites are inappropriate). Failed messages: javax.mail.MessagingE..

https://shanepark.tistory.com/426

**Comment by eeesnghyun** [comment]
감사합니다. 덕분에 빨리 해결했네요 ^^
